tracer/src/Datadog.Trace/AppSec/Coordinator/SecurityCoordinator.Framework.cs

```diff
@@ -283,6 +283,13 @@ internal void ReportAndBlock(IResult? result)
             if (result.ShouldBlock)
             {
                 ChooseBlockingMethodAndBlock(result, reporting, result.BlockInfo, result.RedirectInfo);
+
+                // We should throw an exception to avoid the code from continuing
+
+                if (result.BlockInfo is not null)
```

Comment:
I'm surprised `ChooseBlockingMethodAndBlock` doesn't  do what it says it does i.e. block? 🤔

Reply:
there's actually an edge case for webapi... we actually throw for it, so your subsequent code wont be reached...

Reply:
I have added a comment to cover what would happen in that situation. Thanks!